packages/tentales/src/middleware/default-middlewares.ts

```diff
@@ -8,7 +8,8 @@ import { errorMiddleware } from "./middlewares/error-middleware"
 const bodyParserMiddleware = () => bodyParser()
 
 export const DEFAULT_MIDDLEWARES: Hook[] = [
```

Comment:
```typescript
export const DEFAULT_MIDDLEWARES: Array<HookName, fns[]> = [...]
```

Would it give us anything to type it more strictly?

Reply:
Would `Array<HookName, fns[]>` even work? A hook is not an array though, it just looks an aweful lot like one. A hook as a _tuple_ with the hook name as the first value and a list of middlewares as the second value.

Check the [type declaration of `Hook`](https://github.com/iZettle/tentales/pull/2/commits/163d38879784e9f89f5c6a595d3380e065362520#diff-17970faa687ba972c58a881fcdbe5f75R73), it should be as type safe as we could be (and therefor makes the `Hook[]` above that as well), right?